locations/spiders/sunbelt_rentals_us_ca.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class SunbeltRentalsUsCaSpider(scrapy.Spider):
+    name = "sunbelt_rentals_us_ca"
+    item_attributes = {"brand": "Sunbelt Rentals", "brand_wikidata": "Q102396721"}
```

Comment:
Assuming https://github.com/alltheplaces/alltheplaces/pull/7120 lands...
```suggestion
    item_attributes = {"brand": "Sunbelt Rentals", "brand_wikidata": "Q102396721", "extras": Categories.SHOP_TOOL_HIRE.value}}
```
?